CHANGELOG.md

```diff
@@ -1,58 +1,63 @@
+- Implemented `serde::Serialize` for R types [[#305]](https://github.com/extendr/extendr/pull/305), [[#355]](https://github.com/extendr/extendr/pull/355)
+- `Rany` type and the `as_any` conversion method. [[#320]](https://github.com/extendr/extendr/pull/320)
+- Implemented `std::fmt::Debug` for wrapper types [[#345]](https://github.com/extendr/extendr/pull/345)
+- Implemented the `[[` operator to function on Rust classes, where it functions the same the `$` operator. [[#359]](https://github.com/extendr/extendr/pull/359)
+- `Load` and `Save`, traits that implement the ability to load and save R data to the RDS format. These traits are implemented for all `Robj` [[#363]](https://github.com/extendr/extendr/pull/363)
+- Support for R version 4.2 [[#399]](https://github.com/extendr/extendr/pull/399)
+- The `Dataframe` wrapper struct [[#393]](https://github.com/extendr/extendr/pull/393)
+- The `IntoDataFrame` trait, which can be derived to allow arbitrary Rust structs to be converted to rows of a data frame [[#393]](https://github.com/extendr/extendr/pull/393)
+
+### Changed
+
+- `Strings::elt` now returns an `Rstr` [[#345]](https://github.com/extendr/extendr/pull/345)
+- Renamed `RType` to `Rtype`  [[#345]](https://github.com/extendr/extendr/pull/345)
+- Wrapper types now contain `Robj` fields [[#190]](https://github.com/extendr/extendr/pull/190)
+- The `R!` macro now accepts strings that contain R code. This is now the recommended way of using the macro. [[#203]](https://github.com/extendr/extendr/pull/203)
+- Better error handling for `<&str>::try_from(Robj)` [[#226]](https://github.com/extendr/extendr/pull/226)
+- `SymPair::sym_pair()` now returns `(Option<Robj>, Robj)` [[#225]](https://github.com/extendr/extendr/pull/225)
+- Better error messages when converting Rust integer types to R [[#243]](https://github.com/extendr/extendr/pull/243)
+- `Character` is now called `Rstr` [[#273]](https://github.com/extendr/extendr/pull/273)
+- Bump `ndarray` to 0.15.3. Under [RFC 1977](https://github.com/rust-lang/rfcs/pull/1977) this is a "public dependency" change, and therefore can be considered a breaking change, as consumers of `extendr` that use an older version of `ndarray` will no longer be compatible until they also bump `ndarray` to a compatible version. [[#275]](https://github.com/extendr/extendr/pull/275)
+- `IsNA` trait has been renamed to `CanBeNA` [[#288]](https://github.com/extendr/extendr/pull/288)
+- `list!` has been rewritten, and how returns a `List` struct [[#303]](https://github.com/extendr/extendr/pull/303)
+
+### Deprecated
+- Calling the `R!` macro with non-string types (e.g. `R!(1)`) is now deprecated. [[#203]](https://github.com/extendr/extendr/pull/203)
+
+### Removed
+- `Real`, `Int`, `Bool` and the redundant trait `SliceIter`, which should be replaced with `Doubles`, `Integers`, and `Logicals` respectively [[#304]](https://github.com/extendr/extendr/pull/304), [[#338]](https://github.com/extendr/extendr/pull/338)
```

Comment:
AFAIK, `Bool` was superseded by `Rbool`. I suspect the same applies to `Real` (`Rfloat`) and `Int` (`Rint`), but I am not 100% sure.